train.py

```diff
@@ -83,6 +84,8 @@ def loss_for_permutation(
         # If mask_order is not provided, use the best one
         if input_permutation is None:
             loss += best_loss
+    # Sum -> Average
+    loss /= B
```

Comment:
Does this just work as-is? We don't have to change reduction parameters in any of the above loss functions?

Reply:
We calculate the loss for each single batch in `loss_for_permutation` function and then collect them, so it works.

Reply:
ok.